.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "Bug report \U0001F41B" 
+about: "If something isn't working as expected \U0001F914."
+title: ''
+labels: bug
+assignees: J535D165, qubixes
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug with the front-end is.
+
+**To Reproduce**
+Steps to reproduce the problem:
+
+1.
+2.
+3.
+
+
+```
+your input code here
+```
+
+**Expected behavior**
```

Comment:
Is this one necessary? Should be clear from the bug description.

Reply:
I agree that ideally this should be clear from the bug description. 
I still think it is nice to explicitly ask for providing both "what is wrong" (the bug) and "what you think should have happened ("expected behaviour").

This helps preventing misunderstandings about the functionality of the code. For example, the conversation we had last week about the "--abstract_only" option only made sense after we found out I was assuming the option did something different than it was actually doing.

I removed this separate heading on expected behaviour, and added a sentence on it under the bug description heading, c65c5182bf98a0cefc18ccb125fa50c0cd1a43b8. What do you think?